conf/application.conf

```diff
@@ -78,11 +65,10 @@ microservice {
       authorization-token = "testing-token"
     }
 
-    internal-auth {
```

Comment:
microservice.services.internal-auth is present in qa/staging/production configs. Will need those removing also.